### ISSUE TITLE
Looking for the full changelog - can you please add it?

### ISSUE THREAD

**Issue by xGnoSiSx** [Geniuswas/Anno-2205-Revolutions-Mod-English#5]
The original site is long gone, and now many years later, I can't even remember what the mod did...

can you add the chnagelog from the base mod? (or link to a site that has it) I was unable to find anything in my search..

**Comment by Geniuswas** [comment]
Yeah ill try to find an archive or get a changelog of the original mod.

**Comment by xGnoSiSx** [comment]
Thank you!
